routes.go

```diff
@@ -15,8 +15,9 @@ var listMiddleware = []echo.MiddlewareFunc{middleware.SortAndFilter, middleware.
 var tenancyMiddleware = []echo.MiddlewareFunc{middleware.Tenancy, middleware.LoggerFields, middleware.UserCatcher}
 
 var tenancyWithListMiddleware = append(tenancyMiddleware, listMiddleware...)
-var permissionMiddleware = append(tenancyMiddleware, []echo.MiddlewareFunc{middleware.PermissionCheck, middleware.RaiseEvent}...)
+var permissionMiddleware = append(permissionMiddlewareWithoutEvents, middleware.RaiseEvent)
 var permissionWithListMiddleware = append(listMiddleware, middleware.PermissionCheck)
+var permissionMiddlewareWithoutEvents = append(tenancyMiddleware, middleware.PermissionCheck)
```

Comment:
Aren't `permissionMIddleware` and `permissionMIddlewareWithoutEvents` the same in this case?

Reply:
:facepalm: you're right... I swear I thought I had seen `middleware.RaisEvent` in the `permissionMiddlewareWithoutEvents` and that's why I said it... Sorry! :sweat_smile: